Library/Homebrew/cmd/tap.rb

```diff
@@ -76,6 +80,7 @@ def tap
 
   def force_auto_update?(args:)
     # if no relevant flag is present, return nil, meaning "no change"
-    true if args.force_auto_update?
+    return true if args.force_auto_update?
+    return false if args.no_force_auto_update?
```

Comment:
```suggestion
    if args.force_auto_update?
      true
    elsif args.no_force_auto_update?
      false
    else
      nil
    end
```

Reply:
The `else` clause is removed by `brew style`.